opr/grading.go

```diff
@@ -1,22 +1,24 @@
+package opr
+
 // Copyright (c) of parts are held by the various contributors (see the CLA)
 // Licensed under the MIT License. See LICENSE file in the project root for full license information.
-package opr
 
 import (
 	"encoding/hex"
 	"encoding/json"
-	"github.com/dustin/go-humanize"
 	"sort"
 	"strings"
 	"sync"
 
+	"github.com/dustin/go-humanize"
+
```

Comment:
Since its a housekeeping PR anyway, is there a reason for the blank line here? I noticed it in one or two other files changed, where "errors" and "strings" are separated by a blank line. Figure its just your IDE auto adding the imports in a weird place

Reply:
yeah, all the imports are auto-handled by my IDE (vscode with go-fmt on save). not sure what exactly caused it. i updated it with a fix :smile_cat: